### PR TITLE
Windows bugfixes and some ui for app feed

### PR DESCRIPTION
**Pull request by nk327**
1. Rewrite urls to paths on windows
2. Instead of instantiating scoped sql sessions, use session auto-management to avoid references to stale sessions
3. Clean up sql session when flask request context is popped

### CODE REVIEW

File: applications/hindsight_feed/feeders/browser_history_summary/browser_history_summary.py [hunk 1]
```diff
@@ -26,6 +26,8 @@ def llm_generate(pipeline, prompt, max_tokens):
 else:
     from transformers import AutoTokenizer, AutoModelForCausalLM
 
+    LLM_MODEL_NAME = "meta-llama/Llama-3.2-1B-Instruct"
```
Comment (cparish312) [inline applications/hindsight_feed/feeders/browser_history_summary/browser_history_summary.py:29]: Pull from hindsight_server config now

File: hindsight_android/app/src/main/java/com/connor/hindsight/models/FeedViewModel.kt [hunk 2]
```diff
@@ -0,0 +1,16 @@
+package com.connor.hindsight.models
```
Comment (cparish312) [inline hindsight_android/app/src/main/java/com/connor/hindsight/models/FeedViewModel.kt:1]: If we are going with the Android Element UI then should probably take this out

**Review by cparish312 — COMMENTED** [review]
Awesome thanks for making the feed streaming windows compatible! I think the Android Elements path seems best so should remove that from the PR

File: applications/hindsight_feed/feeders/browser_history_summary/browser_history_summary.py (hunk 1 above)
Comment (nk327) [reply applications/hindsight_feed/feeders/browser_history_summary/browser_history_summary.py:29]: what's a good way to do it? I'm getting an error for importing from beyond toplevel package

Comment (cparish312) [reply applications/hindsight_feed/feeders/browser_history_summary/browser_history_summary.py]: I merged in a fix. I will get with of the manual python path additions soon!

**Pull request merged**